recipe/meta.yaml

```
@@ -18,22 +19,21 @@ requirements:
     - {{ compiler('c') }}
   host:
     - python
-    - cython >=0.29.26,<3
-    - numpy {{ numpy }}
+    - cython >=3.0.10,<4
+    - numpy >=1.23
```

```suggestion
    - numpy 1.23 # [py<311]
    - numpy {{ numpy }}
```
So that pin_compatible in run yields the broadest range.

---

@cbouss it seems this way is causing some issues